build.sbt

```diff
@@ -32,12 +32,39 @@ lazy val dependencySettings =
  * ==================================== tasks =====================================
  */
 
+lazy val stdLibArtifact = Artifact("stdlib", "jar", "jar")
+
+lazy val packStdLib = TaskKey[File]("packStdLib", "Packs the std-lib into a single jar.")
+
+packStdLib := {
+  val rootDir = file(".")
+  val (_, coreFile) = packagedArtifact.in(Compile, packageBin).value
+
+  val targetDir = coreFile.getParentFile()
+
+  def filesIn(dir: File): Seq[File] = {
+    val (files, dirs) = IO.listFiles(dir).partition(_.isFile)
+    files.filter(_.getName endsWith ".mgn") ++ dirs.flatMap(filesIn)
+  }
+
+  def relativeTo(base: File, f: File): String =
+    base.toURI.relativize(f.toURI).getPath
+
+  val stdLibJar = filesIn(rootDir / "std").map(f => f -> relativeTo(rootDir, f))
+  val scalaMajorV = scalaVersion.value.split("\\.").init.mkString(".")
+  val stdLibJarFile = targetDir / s"${name.value}-stdlib_$scalaMajorV-${version.value}.jar"
+
+  IO.jar(stdLibJar, stdLibJarFile, new java.util.jar.Manifest())
+  stdLibJarFile
+}
+
 lazy val build = TaskKey[Unit]("build", "Builds morganey and copies all generated files into the target folder.")
 
 build := {
   val (_, coreFile) = packagedArtifact.in(Compile, packageBin).value
   val (_, macroFile) = packagedArtifact.in(macros).in(Compile, packageBin).value
   val (_, kernelFile) = packagedArtifact.in(kernel).in(Compile, packageBin).value
+  val _ = packStdLib.value
```

Comment:
The assignment in this statement can safely be replaced by the method call (without assignment).

```scala
packStdLib.value
```